skyportal/tests/api/test_sources.py

```diff
@@ -52,6 +52,69 @@ def test_token_user_retrieving_source_with_phot(view_only_token, public_source):
     )
 
 
+def test_token_user_retrieving_source_with_thumbnails(view_only_token, public_source):
+    status, data = api(
+        "GET",
+        f"sources/{public_source.id}",
+        params={"includeThumbnails": "true"},
+        token=view_only_token,
+    )
+    assert status == 200
+    assert data["status"] == "success"
+    assert all(
+        k in data["data"]
+        for k in ["ra", "dec", "redshift", "dm", "created_at", "id", "thumbnails"]
+    )
+
+
+def test_token_user_retrieving_source_without_nested(
+    view_only_token, public_group, upload_data_token
+):
+    obj_id1 = str(uuid.uuid4())
+    obj_id2 = str(uuid.uuid4())
+    status, data = api(
```

Comment:
What's the motivation for creating these sources via the API as opposed to using the fixtures as in the above test?

Reply:
When I was running the test without creating these sources, the queried list of sources was sometimes empty